toolchain/check/testdata/packages/raw_core.carbon

```diff
@@ -0,0 +1,275 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/packages/raw_core.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/packages/raw_core.carbon
+
+// --- package_raw_core.carbon
+
+package r#Core;
```

Comment:
Just to be sure, did you notice #4900 going by?

Reply:
Yes, thanks. Looks like whichever one of these two lands second will need to update that test to not expect a failure.